source/Projectile.cpp

```diff
@@ -82,6 +85,8 @@ Projectile::Projectile(const Projectile &parent, const Point &offset, const Angl
 	hitsRemaining = weapon->PenetrationCount();
 
 	cachedTarget = TargetPtr().get();
+	if(cachedTarget)
+		targetDisabled = cachedTarget->IsDisabled();
```

Comment:
This should be using the `targetDisabled` value of the parent.

Reply:
What do you mean? I thought you wanted me to cache whether the target was disabled, and if it changes to disabled, break the target. I don't think how the parent comes into that.

Reply:
It's cause this is the constructor for a submunition projectile. I meant the parent projectile, so like `targetDisabled = parent.targetDisabled;` (like it's done for the targetGovernment too).

Reply:
aaaah that makes sense